Application/Source/Detail/View/DetailViewModel.swift

```diff
@@ -50,24 +48,34 @@ class DetailViewModel: ViewModel, SelectionPresentingViewModel {
                 .disposed(by: self.disposeBag)
         })
 
-    init(selectionFactory: SelectionViewModelFactoryProtocol) {
-        self.selectionFactory = selectionFactory
+    typealias Factory = SelectionViewModelFactoryProtocol & FoodInfoViewModelFactoryProtocol
+
+    init(foods: Property<[Food]>, factory: Factory) {
+        self.foods = foods
+        self.factory = factory
     }
 
     private let selectionResultRelay = BehaviorRelay<String?>(value: nil)
-    private let selectionFactory: SelectionViewModelFactoryProtocol
+    private let factory: Factory
     private let disposeBag = DisposeBag()
 
 }
 
-protocol DetailViewModelFactoryProtocol: SelectionViewModelFactoryProtocol {
+protocol DetailViewModelFactoryProtocol: SelectionViewModelFactoryProtocol, FoodInfoViewModelFactoryProtocol {
+    var foods: Property<[Food]> { get }
+
     func makeDetailViewModel() -> DetailViewModel
 }
 
 extension DetailViewModelFactoryProtocol {
 
+    // This can be backed by a datastore. For now, use a computed property to represent persistent data
+    var foods: Property<[Food]> {
```

Comment:
Perhaps this should just be defined on the class that implements `DetailViewModelFactoryProtocol` (`DetailNavigationModelFactory`). Even though this is temporary, it's odd that it would instantiate the data model in a protocol extension. It could look like this (in `DetailNavigationModel.swift`):
```
class DetailNavigationModelFactory: DetailNavigationModelFactoryProtocol {
    let foods = Property([.beans, .greens, .potatoes, .tomatoes])
}
```
I believe one of the Stubs would also need to be updated.